requirements/travis.txt

```diff
@@ -11,7 +11,7 @@ coverage==4.5.2           # via codecov
 filelock==3.0.10          # via tox
 idna==2.7                 # via requests
 pluggy==0.8.0             # via tox
-psycopg2==2.7.6.1
+psycopg2>=2.8.4
```

Comment:
@codekiln, per [this comment](https://github.com/psycopg/psycopg2/issues/854#issuecomment-611791946) we'll likely need to update the correct files and `pip-compile` again